homeassistant/components/emoncms/config_flow.py

```diff
@@ -196,10 +201,7 @@ async def async_step_init(
         if user_input:
             include_only_feeds = user_input[CONF_ONLY_INCLUDE_FEEDID]
             return self.async_create_entry(
-                title=sensor_name(url),
                 data={
-                    CONF_URL: url,
-                    CONF_API_KEY: api_key,
```

Comment:
The option flow doesn't give the ability to change these two properties - and there is no reason to keep them stored in the options.
I have adjusted the code to always read them from the entry data.

Reply:
OK understood, it looks better like that...